tests/open-ce-tests.yaml

```diff
@@ -0,0 +1,32 @@
+tests:
+  - name: Setup Apache Arrow Tests
+    command: |
+        PYARROW_LOCATION=$(python -c "import os; import pyarrow; print(os.path.dirname(pyarrow.__file__))")
+        echo $PYARROW_LOCATION
+        conda install -y git
+        git clone -b apache-arrow-$(python -c "import pyarrow; print(pyarrow.__version__)") http://github.com/apache/arrow
+        cp -r arrow/python/pyarrow/tests $PYARROW_LOCATION
+        conda install -y hypothesis pytest pytz numpy
+        pip install pytest-lazy-fixture
+        pushd arrow/testing
+        export ARROW_TEST_DATA=$(pwd)
+        popd
+  - name: Test Imports
+    command: |
+        python -c "import pyarrow" 
+        python -c "import pyarrow.dataset" 
+        python -c "import pyarrow.flight" 
+        python -c "import pyarrow.orc" 
+        python -c "import pyarrow.parquet" 
+        python -c "import pyarrow._hdfs" 
+
+  - name: Test Imports - CUDA
```

Comment:
This test should be wrapped around an if statement ensuring that the build_type is cuda.